apps/meteor/client/sidebarv2/RoomList/RoomList.tsx

```diff
@@ -66,14 +44,27 @@ const RoomList = () => {
 	usePreventDefault(ref);
 	useShortcutOpenMenu(ref);
 
-	const { groupCounts, groupList, roomList } = getRoomsByGroup(roomsList);
+	const handleCollapse = (groupTitle: string) => {
+		if (groupTitle) {
+			handleCollapsedGroups(groupTitle);
+		}
+	};
 
 	return (
 		<Box position='relative' display='flex' overflow='hidden' height='full' flexGrow={1} flexShrink={1} flexBasis='auto' ref={ref}>
 			<GroupedVirtuoso
-				groupCounts={groupCounts}
-				groupContent={(index) => <SidebarV2GroupTitle title={t(groupList[index])} />}
-				itemContent={(index) => <RoomListRow data={itemData} item={roomList[index]} />}
+				groupCounts={groupsCount}
+				groupContent={(index) => (
+					<SidebarV2CollapseGroup
```

Comment:
I also noticed that the collapse arrow sometimes don't animate for me. 
_(In the gif I'm clicking once in each section)_

![sidebar-colapse-gif](https://github.com/user-attachments/assets/2dc01af7-8b26-4e01-95dc-f7ae767b5f96)

Reply:
here too: `yarn && yarn build` should fix that